crates/primitives/src/transaction/access_list.rs

```diff
@@ -51,6 +51,21 @@ impl AccessList {
         self.flatten().collect()
     }
 
+    /// Converts the ['AccessList'] into a Vec of (Address, `Vec<U256>`), as expected by revm
+    pub fn to_ref_slice(&self) -> Vec<(Address, Vec<U256>)> {
```

Comment:
this does not return a slice

maybe `fn as_flattened`?

Reply:
agree w/ `as_flattened`, I would also remain the other function to `into_flattened` and ensure `as_flattened` returns an iterator instead of collecting